pkg/kor/delete.go

```diff
@@ -45,9 +45,12 @@ func DeleteResourceCmd() map[string]func(clientset kubernetes.Interface, namespa
 		"PDB": func(clientset kubernetes.Interface, namespace, name string) error {
 			return clientset.PolicyV1beta1().PodDisruptionBudgets(namespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
 		},
-		"Role": func(clientset kubernetes.Interface, namespace, name string) error {
+		"Roles": func(clientset kubernetes.Interface, namespace, name string) error {
 			return clientset.RbacV1().Roles(namespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
 		},
+		"ClusterRoles": func(clientset kubernetes.Interface, namespace, name string) error {
```

Comment:
```suggestion
		"ClusterRole": func(clientset kubernetes.Interface, namespace, name string) error {
```

The resource name must match the resource name passed in `clusterroles.go` func GetUnusedClusterRoles